docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -121,6 +121,7 @@ Note that the first part of the timestamp is in UTC (same thing as GMT). The tim
 - Presto
 - SparkSQL
 - SQLite
+- Metabase Sample Database
```

Comment:
Should we mention that it's H2?

Reply:
H2 was removed from this list in #33019 so I figured I shouldn't add it back